setup.py

```diff
@@ -85,7 +85,7 @@ def get_package_data(parent, roots):
 
 setup(
     name="pyfarm.agent",
-    version="0.8.3",
+    version="0.8.3.1",
```

Comment:
Yeah I think keeping the versioning scheme consistent with prior history and other packages would be best.

If it makes it easier for you, I'll be happy to release 0.8.3, merge this and then release 0.8.4.  That will make the development version 0.8.5 which might help with your tracking problem?